internal/usage/usage.go

```diff
@@ -36,8 +36,8 @@ const (
 	DBUsername                      = "Username for authenticating to MongoDB."
 	TeamName                        = "Name of the team."
 	UserID                          = "The ID of the user."
-	Hostname                        = "The hostname or IP address of the LDAP server."
-	Port                            = "The port to which the LDAP server listens for client connections. Default: 636"
+	LDAPHostname                    = "The hostname or IP address of the LDAP server."
+	LDAPPort                        = "The port to which the LDAP server listens for client connections. Default: 636"
```

Comment:
[q] Will changing the parameter name have any effect on existing clients? I'm wondering are we breaking parameters for users here.